hook-common/src/kafka_messages/plugin_logs.rs

```diff
@@ -0,0 +1,130 @@
+use chrono::{DateTime, Utc};
+use serde::{Serialize, Serializer};
+use uuid::Uuid;
+
+use super::{serialize_datetime, serialize_uuid};
+
+#[allow(dead_code)]
+#[derive(Serialize)]
+pub enum PluginLogEntrySource {
+    System,
+    Plugin,
+    Console,
+}
+
+#[allow(dead_code)]
+#[derive(Serialize)]
+pub enum PluginLogEntryType {
+    Debug,
+    Log,
+    Info,
+    Warn,
+    Error,
+}
+
+#[derive(Serialize)]
+pub struct PluginLogEntry {
+    #[serde(serialize_with = "serialize_source")]
+    pub source: PluginLogEntrySource,
+    #[serde(rename = "type", serialize_with = "serialize_type")]
+    pub type_: PluginLogEntryType,
+    #[serde(serialize_with = "serialize_uuid")]
+    pub id: Uuid,
+    pub team_id: u32,
+    pub plugin_id: u32,
+    pub plugin_config_id: u32,
```

Comment:
The main reason we developed the generic `log_entries` was because of these fields: not every system that wants to write logs has a numerical id associated with it, and generating one could risk colliding with existing plugins.

However, like we discussed, for webhooks we will have an associated plugin, so using plugin_log_entries as our log table is possible.